scripts/amplitude/wrapper.js

```diff
@@ -0,0 +1,71 @@
+import { configFile } from "../utils/configFile.js";
+import { AMPLITUDE_API_KEY, OPT_IN_NAME } from "./config.js";
+import { init, track, Identify, identify } from "@amplitude/analytics-node";
+import { currentUser } from "../utils/user.js";
+
+class AmplitudeWrapper {
+  get userType() {
+    // TODO: Implement once we have the data available in the UserSerializer
```

Comment:
Just doubling checking that there's a ticket for this further down the road.

Reply:
I don't think there's one yet, but this was defined as a mandatory property in the spreadsheet, so I added this to not forget. I will ask Bryan, probably should create the ticket now